content/docs/guides/contribute/review-process.en.md

```diff
@@ -0,0 +1,20 @@
+Here's a suggested workflow. The quotation marks indicate the **necessity** of communication to ensure the smooth flow of PR validation:
+
+```mermaid
+sequenceDiagram
+  actor A as PR author
+  actor R as Reviewer/Maintainer
+  
+  A->>R: ASKS "I need a review"
```

Comment:
```suggestion
  A->>R: ASKS for a review
```

Reply:
I prefer explicitly write down "I need a review" as "It's approved". Those are things that MUST be clearly said/written IMO.

Reply:
Both ways are clear and explicit. I find quoting a fictional conversation odd in this context: most technical sequence diagrams [don't use fictional conversations to describe protocols](https://duckduckgo.com/?t=ffab&q=oidc+sequence+diagram&iax=images&ia=images), but instead:

- either describe action performed at each step: `request a review`
- or describe the message itself: `review request`

Reply:
Nevertheless, I prefer to keep it explicit, in fictional conversation, not actually based on what is done in general, but on the working experience we have within our team where the interpretation of what a "review request" should be in reality. No interpretation here, just a clear & specific message.